Src/ILGPU.Algorithms/PTX/PTXMath.cs

```diff
@@ -185,6 +185,11 @@ public static float Floor(float value)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static double Asin(double value)
         {
+            if (value == 1)
```

Comment:
can you mark double and float constants with a `.0`/`.0f`explicitly?

Reply:
Done